src/AcmForm/AcmForm.tsx

```diff
@@ -21,9 +21,11 @@ export function AcmForm(props: FormProps) {
     const [validate, setValidate] = useState(false)
     const [errors, setErrors] = useState<{ [id: string]: string | undefined }>({})
     const setError = (id: string, error?: string) => {
-        const copy = { ...errors }
-        copy[id] = error
-        setErrors(copy)
+        setErrors((prevState) => {
+            const copy = { ...prevState }
+            copy[id] = error
+            return copy
```

Comment:
```js
setErrors(prevState => ({ ...prevState, [id]: error }))
```
?

Reply:
This was a fix... since all the errors are set asynchronously, the setting of errors was stepping on each other and overwritting. This uses a function to update while preserving the previous state.

Reply:
Yeah just a suggested condensed version of the changes